applications/StatisticsApplication/python_scripts/spatial_statistics_process.py

```diff
@@ -39,23 +40,38 @@ def __init__(self, model, settings):
 
         default_parameters = Kratos.Parameters("""
         {
-            "model_part_name" : "PLEASE_SPECIFY_MODEL_PART_NAME",
+            "model_part_name"         : "PLEASE_SPECIFY_MODEL_PART_NAME",
+            "echo_level"              : 0,
+            "computation_processes"   : [],
+            "computation_points"      : [
+                "ExecuteInitialize",
+                "ExecuteInitializeSolutionStep",
+                "Execute",
+                "ExecuteFinalizeSolutionStep"
+            ],
```

Comment:
I think we should specify `Execute` as unique computation point by default (not 100% sure).

Reply:
I am not sure about this since some processes have defined multiple steps. Just in case some one uses this with some other process and they don't know the exact computation points. I left the flexibility in there also.

Reply:
Fair enough. My point was to promote the use of  `Execute` processes only, but at this point this is probably way too strict.